sql/src/test/java/io/crate/execution/engine/join/HashInnerJoinBatchIteratorMemoryTest.java

```diff
@@ -79,21 +85,7 @@ public void testReleaseAccountingRows() throws Exception {
         TestingRowConsumer consumer = new TestingRowConsumer();
         consumer.accept(it, null);
         consumer.getResult();
-        assertThat(leftIterator.countCallsForReleaseMem, is(7));
-    }
-
-    private class TestRamAccountingBatchIterator extends RamAccountingBatchIterator<Row> {
-
-        private int countCallsForReleaseMem = 0;
-
-        private TestRamAccountingBatchIterator(BatchIterator<Row> delegatePagingIterator, RowAccounting rowAccounting) {
-            super(delegatePagingIterator, rowAccounting);
-        }
-
-        @Override
-        public void releaseAccountedRows() {
-            countCallsForReleaseMem++;
-            super.releaseAccountedRows();
-        }
+        verify(rowAccounting, times(8)).release();
```

Comment:
Increased from 7 to 8 because now it is called initially also once in the CTOR. Shouldn't have too much impact.